docs/source/smartdevice.rst

```diff
@@ -45,6 +45,30 @@ Simple example script showing some functionality:
     if __name__ == "__main__":
         asyncio.run(main())
 
+If you are connecting to a newer KASA or TAPO device you can get the device via discovery or connect directly with DeviceConfig:
+
+.. code-block:: python
+
+    import asyncio
+    from kasa import Discover, Credentials
+
+    async def main():
+        device = await Discover.discover_single(
+            "127.0.0.1",
+            credentials=Credentials("myusername", "mypassword"),
+            discovery_timeout=10
+        )
+
+        config = device.config # DeviceConfig.to_dict() can be used to store for later
+
+        # To connect directly later without discovery
+
+        later_device = await SmartDevice.Connect(config=config)
```

Comment:
```suggestion
        later_device = await SmartDevice.connect(config=config)
```
I think we should probably "externalize" these examples into regular `.py` files to allow testing them, as I'm worried about documentation rot over time. This can be done in a separate PR at some later point though, so just make sure the example script is working for now.

Reply:
Done